inst/extdata/PA2022CH_en_exec_summary/template.Rmd

```diff
@@ -260,9 +260,13 @@ plot_alignment_table(data_alignment_table)
 
 \newpage
 
-```{r conditional_print, child='survey-chapter.Rmd', eval = (dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
+```{r survey_chapter, child='survey-chapter.Rmd', eval = (dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
 ```
 
+```{r survey_chapter, child='survey-chapter-alternate.Rmd', eval = !(dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
```

Comment:
```suggestion
```{r survey_chapter_alternate, child='survey-chapter-alternate.Rmd', eval = !(dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
```

I think this might give a duplicate chunk error?

Reply:
very possible, changed